AISKU/Tests/Unit/src/AISKUSize.Tests.ts

```diff
@@ -1,14 +1,33 @@
 import { AITestClass, Assert } from "@microsoft/ai-test-framework";
+import { strUndefined } from "@microsoft/applicationinsights-core-js";
 import * as pako from "pako";
+import { Snippet } from "../../../src/Initialization";
 
 export class AISKUSizeCheck extends AITestClass {
-    private readonly MAX_DEFLATE_SIZE = 40;
+    private readonly MAX_DEFLATE_SIZE = 44;
```

Comment:
This I guess would go down after minification work!  Should we create a task and reference here?

Reply:
Yes, that is the plan -- still working on these so I don't have final numbers yet. But I plan to reduce once all of the minifications are checked in.

It does appear that Core will be larger as that is where most of the changes are located, but as I'm not using namespaces tree-shaking will remove anything unused.